app/models/miq_alert.rb

```diff
@@ -1,6 +1,13 @@
 class MiqAlert < ApplicationRecord
   include UuidMixin
 
+  AVAILABLE_SEVERITIES = {
+    nil      => nil,
+    :info    => "Information",
+    :warning => "Warning",
+    :error   => "Error"
```

Comment:
Is this a presentation string, or is this the value store in the database?  If the former, I would expect the lookup to be the UI's domain, and if the latter, I would expect it to be downcased.

Reply:
@Fryguy Yes, that was the intention. I worked that out with @dclarizio. Dan, can this be move over to the UI side?